astro/src/content/docs/lifecycle/authenticate-users/integrations/saml/pagerduty.mdx

```diff
@@ -43,22 +44,22 @@ You can review our [Technical Support documentation](/docs/operate/troubleshooti
 
 ### In PagerDuty
 
-1. The _Account Owner_ can configure the SSO settings by navigating to <strong>User Icon -> Account Settings -> Single Sign-On</strong>.
+1. The _Account Owner_ can configure the SSO settings by navigating to <Breadcrumb>User Icon -> Account Settings -> Single Sign-On</Breadcrumb>.
 2. Select the **SAML** radio button.
```

Comment:
Because it's in pagerduty? https://support.pagerduty.com/docs/sso, from what i understand this counts as a Breadcrumb "If you can click on something and go somewhere (like a tab)"